### DISCUSSION TITLE
support springboot

### DISCUSSION THREAD

**Discussion by livk-cloud** [burningwave/core#12] [Ideas]
have springboot starter jar package？
Many configurations require the reuse of properties, which is cumbersome

**Comment by Roberto-Gentili** [comment]
Hello, I moved this message to discussions because it is not an issue but a request for new feature.
At the moment you can change the name of the burningwave.static.properties and burningwave.properties files to application.properties (yml format is not supported) through the methods:
* [org.burningwave.core.assembler.ComponentContainer.Configuration.Default.setFileName](https://github.com/burningwave/core#configuration)
* [org.burningwave.core.assembler.StaticComponentContainer.Configuration.Defaiult.setFileName](https://github.com/burningwave/core#configuration-1)

I'm sorry but I have a lot of software to mantain at the moment and I am unable to create a new project to integrate the Core configuration with Spring configuration, but I'll try to make the Core able to read the yml format only when it will be used with Spring

**Comment by Roberto-Gentili** [comment]
... I just completed the necessary changes to facilitate the integration that I indicated in the previous comment: now to configure Burningwave Core you can use the application.yml file (without calling the * .setFileName methods)  by following the steps indicated into the [post in the help guide](https://www.burningwave.org/forum/topic/how-can-i-integrate-the-configuration-properties-in-spring/#postid-102).

**Comment by livk-cloud** [comment]
ok,thank you
